.github/scripts/tests/test-gas-big-obj.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+GAS_PATH="$TOOLCHAIN_PATH/bin/$TARGET-as"
+OBJDUMP_PATH="$TOOLCHAIN_PATH/bin/$TARGET-objdump"
+
+echo "::group::Test GAS big object PE/COFF format"
+
+    cd $ROOT_PATH/tests
+
+    EXIT_CODE=0
+    $GAS_PATH big-obj.s -o big-obj.o || EXIT_CODE=$?
+
+    if [ $EXIT_CODE -eq 0 ]; then
+        echo "Unexpected success, aarch64-w64-mingw32-as should default to small object PE/COFF format"
```

Comment:
Both options are needed. Internal tests will help to prepare patches for upstreaming and detect regression faster. When there is some fix which should be upstreamed the internal test is a good way for quick validation. However as it is mentioned just internal test is enough for now.

Reply:
JFYI, I am not questioning the need of existence of both test suites. I am quesitoning the suitability of the testing framework we are using for the internal tests.

Reply:
One more suggestion: it would be great to trigger this binutils test if possible directly from binutils without implementing it.

Reply:
With the way how our CI is structured, this is not easily doable. In `advanced.yml` the toolchain is built in `build-toolchain` step and only toolchain and its runtime is packaged and stored in the artifacts. Then we do testing in a separate step `build-aarch64-tests` resp. `execute-aarch64-tests` where we only download toolchain and runtime, so the build folder with bintuils built tests is not available.

So to reuse the test from binutils one of the following would need to be done:
1. Either store `build` folder from `build-toolchain` in artifacts
2. Or run the test directly in `build-toolchain` step

To me both of these options seem too expensive or invasive for implementing one test. What do you think?

Reply:
IIRC, no binutils test is actually executing the target executable, just comparing the outputs, so it's completely fine to run the binutils tests within the build job.

Reply:
I was thinking that it will be weird design if a build job can fail because a test there failed but if that's not a concern, I'll move it to the `build-toolchain` step. It will be the easiest implementation-wise.

Reply:
I think the contrary - the faster we can get feedback that something was broken the better. Which brings me to idea that the binutils tests could be executed right after binutils have been built before even start building stage1 GCC.

Reply:
Moved directly into the binutils build step because that's the only place where we have `build` folder available.